0283_MoveZeroes.py

```diff
@@ -0,0 +1,19 @@
+class Solution(object):
+    def moveZeroes(self, nums):
+        nonzeroi=len(nums)-1
```

Comment:
When you define a variable which has words, like this one "nonzeroi -> "non", "zero", "i', try to separate those words in some way, like "non_zero_i" or "nonZeroI". For example, take a look at https://en.wikipedia.org/wiki/Camel_case